syslogish/server.go

```diff
@@ -1,27 +1,31 @@
 package syslogish
 
 import (
-	"encoding/json"
 	"fmt"
 	"log"
 	"net"
 	"strings"
+
+	"github.com/deis/stdout-metrics/influx"
+	"github.com/deis/stdout-metrics/util"
+	"github.com/influxdata/influxdb/client/v2"
 )
 
 const (
 	bindHost  = "0.0.0.0"
 	bindPort  = 1514
-	queueSize = 500
+	queueSize = 50000
```

Comment:
not blocking for this PR - worth making this and/or the above params configurable?